lib/notify/notification.rb

```diff
@@ -83,6 +84,19 @@ def send_changed_password_email(email_address:, first_name:)
     Rails.logger.error(e.message)
   end
 
+  def send_admin_changed_user_password_email(email_address:, first_name:, reset_url:)
+    mail_client.send_email(
+      email_address: email_address,
+      template_id: ADMIN_RESET_USER_PASSWORD_TEMPLATE,
+      personalisation: {
+        first_name: first_name,
+        reset_url: "[#{url}/reset-user-password](#{reset_url})",
```

Comment:
I wonder if this would work?
```suggestion
        reset_url: "[#{url}/#{reset_password_path}](#{reset_url})",
```
so we could get rid of the hardcoded values completely?